### ISSUE TITLE
Properties menu does not like capital letters.

### ISSUE THREAD

**Issue by DudeThatsErin** [gfxholo/iconic#24]
<img width="1129" alt="Screenshot 2024-09-07 at 1 30 00 PM" src="https://github.com/user-attachments/assets/624f3a9b-2382-4810-b766-c2ccfaca163d">
<img width="1085" alt="Screenshot 2024-09-07 at 1 30 52 PM" src="https://github.com/user-attachments/assets/db3a77cd-9009-4a44-8056-86172ffd790e">

I have recreated this property a few times and the field above the icon search always comes up blank. I believe it is due to the capital letter in the property because if I create a new property without the capital letter, it works. I was trying to set the icon of `homeAddress`. Remove the capital letter and it works.

**Comment by gfxholo** [comment]
Oh wow, that's a subtle bug. I found the code that's causing it too. You can work around this for now by setting `homeAddress` from the Properties sidebar, because the bug doesn't exist over there.

You're a developer though, so feel free to patch it yourself by opening `main.js` x)

 Find:
```js
    const propBase = this.app.metadataTypeManager?.properties[propId] ?? {};
```

Replace:
```js
    const propBases = Object.values(this.app.metadataTypeManager?.properties) ?? [];
    const propBase = propBases.find(propBase => propBase.name === propId) ?? {};
```

**Comment by DudeThatsErin** [comment]
Thanks! Will do!

**Comment by DudeThatsErin** [comment]
@gfxholo I just realized when you publish this, it creates a `main.js` file that is compiled so I can't update that. I will await your next plugin update. 🥰

**Comment by gfxholo** [comment]
@DudeThatsErin No! That sucks. This will definitely be fixed in `1.1`, but I might push out an earlier patch since this could be aggravating quite a few people.

Thanks a bunch for the quality bug report 💜
